config/main.py

```diff
@@ -97,10 +97,11 @@
 
 CFG_PORTCHANNEL_PREFIX = "PortChannel"
 CFG_PORTCHANNEL_PREFIX_LEN = 11
-CFG_PORTCHANNEL_NAME_TOTAL_LEN_MAX = 15
 CFG_PORTCHANNEL_MAX_VAL = 9999
 CFG_PORTCHANNEL_NO="<0-9999>"
 
+IFNAMSIZ = 16
```

Comment:
@arfeigin why not to import from `utilities_common.helper`?

Reply:
fixed